.circleci/config.yml

```diff
@@ -218,8 +218,7 @@ jobs:
     working_directory: /go/src/github.com/3scale/3scale-operator
     steps:
       - setup_remote_docker:
-          reusable: true
-          exclusive: false
```

Comment:
https://circleci.com/docs/2.0/docker-layer-caching/#remote-docker-environment

exclusive is not used anymore (and it is true always now), and reusable has been renamed to docker_layer_caching